docs/_docs/upgrading.md

```diff
@@ -8,3 +8,5 @@ Upgrading from an older version of Jekyll? Upgrading to a new major version of J
 
 - [From 0.x to 1.x and 2.x](/docs/upgrading/0-to-2/)
 - [From 2.x to 3.x](/docs/upgrading/2-to-3/)
+
+If you are making a minor update (for example from 3.3.1  to 3.3.2 ) run 'gem update jekyll' when in your site directory. 
```

Comment:
minor nit, but could you take out some of the spaces so that it looks like `from 3.3.1 to 3.3.2)`?